go_covert_lib/controller/processor/symmetricEncryption/symmetricEncryption.go

```diff
@@ -0,0 +1,237 @@
+package symmetricEncryption
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/des"
+	"errors"
+)
+
```

Comment:
If the message is more than 255 characters this will fail. The reason I said make it 8 bytes is to allow for a full 
64 bit int (the largest size in go, and much more room than could feasibly be needed).